src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url("../images/star.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+
+    &:nth-child(-n+4) {
+      background-image: url("../images/star-active.svg");
+    }
```

Comment:
keep it optional, what if I will change modifier to other number?
12'th point in checklist